openedx/features/caliper_tracking/utils.py

```diff
@@ -4,7 +4,7 @@
 from dateutil.parser import parse
 
 UTC_DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%S.%f'
```

Comment:
Constant should be declared after the import statements. kindly correct this.

Reply:
done